src/plugins/mf_priority.cpp

```diff
@@ -298,11 +477,31 @@ static int validate_cb (flux_plugin_t *p,
     if (flux_jobtap_job_aux_set (p,
                                  FLUX_JOBTAP_CURRENT_JOB,
                                  "mf_priority:bank_info",
-                                 &bank_it->second,
+                                 b,
                                  NULL) < 0)
         flux_log_error (h, "flux_jobtap_job_aux_set");
 
-    bank_it->second.cur_active_jobs++;
+    b->cur_active_jobs++;
+
+    return 0;
+
+no_user_info:
+    b = &users[userid]["DNE"];
+    users_def_bank[userid] = "DNE";
+
+    b->fairshare = 0.1;
+    b->max_run_jobs = BANK_INFO_MISSING;
+    b->cur_run_jobs = 0;
+    b->max_active_jobs = 0;
+    b->cur_active_jobs = 0;
+    b->held_jobs = std::vector<long int>();
+
+    if (flux_jobtap_job_aux_set (p,
+                                 FLUX_JOBTAP_CURRENT_JOB,
+                                 "mf_priority:bank_info",
+                                 b,
+                                 NULL) < 0)
+        flux_log_error (h, "flux_jobtap_job_aux_set");
```

Comment:
I think I saw this `no_user_info` block twice? Maybe this could/should be a function?